internal/descriptor/descriptor.go

```diff
@@ -37,6 +50,16 @@ func ShortDigest(desc ocispec.Descriptor) (digestString string) {
 	return digestString
 }
 
+// Plain returns a plain descriptor that contains only MediaType, Digest and
+// Size.
+func Plain(desc ocispec.Descriptor) ocispec.Descriptor {
```

Comment:
This function seems to be copied from `oras-go`. Can you add a reference in the comment block?